pom.xml

```diff
@@ -781,31 +774,25 @@
     <profile>
       <!-- This profile is used to enable GraalVM native image testing -->
       <id>native</id>
-
       <dependencies>
-
         <dependency>
-          <groupId>org.junit.vintage</groupId>
-          <artifactId>junit-vintage-engine</artifactId>
-          <version>5.9.2</version>
-          <scope>test</scope>
-        </dependency>
-
-        <dependency>
-          <groupId>org.graalvm.buildtools</groupId>
-          <artifactId>junit-platform-native</artifactId>
-          <version>0.9.20</version>
-          <scope>test</scope>
+          <groupId>org.opentest4j</groupId>
```

Comment:
This dependency is needed to resolve class `org.opentest4j.TestAbortedException`

Error log without this dependency:
```
Failures (1):
  JUnit Jupiter
    => org.junit.platform.commons.JUnitException: TestEngine with ID 'junit-jupiter' failed to execute tests
       org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:113)
       org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
       org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
       org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
       org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
       [...]
     Caused by: java.lang.NoClassDefFoundError: org.opentest4j.TestAbortedException
       org.junit.jupiter.engine.support.OpenTest4JAndJUnit4AwareThrowableCollector.<clinit>(OpenTest4JAndJUnit4AwareThrowableCollector.java:38)
       org.junit.jupiter.engine.support.JupiterThrowableCollectorFactory.createThrowableCollector(JupiterThrowableCollectorFactory.java:34)
       org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:89)
       org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
       org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
       [...]
```

Full log: https://github.com/googleapis/java-shared-config/actions/runs/4520881632/jobs/7962275716